src/main/scala/uk/gov/hmrc/ct/computations/calculations/SBACalculator.scala

```diff
@@ -34,16 +34,17 @@ trait SBACalculator extends NumberRounding with AccountingPeriodHelper {
   def isEarliestWrittenContractAfterAPStart(contractDate: LocalDate, apStartDate: LocalDate): Boolean = contractDate.isAfter(apStartDate)
 
   /* This is the 2% rounded up*/
-  def getAmountClaimableForSBA(apStartDate: LocalDate, apEndDate: LocalDate, contractDate: Option[LocalDate], cost: Option[BigDecimal]): Option[Int] = {
+  def getAmountClaimableForSBA(apStartDate: LocalDate, apEndDate: LocalDate, contractDate: Option[LocalDate], firstUsage: Option[LocalDate], cost: Option[BigDecimal]): Option[Int] = {
 
     val daysInTheYear = getDaysIntheYear(apStartDate)
 
-    (contractDate, cost) match {
-      case (Some(maybeContractDate), Some(mayBeCost)) => {
-        val dailyRate = apportionedCostOfBuilding(mayBeCost, daysInTheYear)
+    (contractDate, firstUsage, cost) match {
+      case (Some(maybeContractDate), Some(maybeFirstUsgae), Some(maybeCost)) => {
```

Comment:
if this is wrapped in a Some, they won't be 'maybeContractDate'...the maybe prefix isn't needed.